test/test_circular_buffer.jl

```diff
@@ -5,7 +5,8 @@ using Base.Test
 cb = CircularBuffer{Int}(5)
 @test length(cb) == 0
 @test capacity(cb) == 5
-@test_throws ArgumentError first(cb)
+# throws ArgumentError on v0.4 and BoundsError on v0.5
+@test_throws Exception first(cb)
```

Comment:
run contrib/commit-name.sh on the merge commit https://github.com/JuliaLang/julia/commit/efa23adbbefca173290c3fafe35ebde8e7f8fe6a